ios/browser/api/web_image/web_image.mm

```diff
@@ -34,13 +37,11 @@ - (instancetype)initWithBrowserState:(ChromeBrowserState*)browserState {
 }
 
 - (void)downloadImage:(NSURL*)url
-         maxImageSize:(NSUInteger)maxImageSize
-       animationDelay:(CGFloat)animationDelay
-           completion:(void (^)(UIImage* image,
+           completion:(void (^)(UIImage* _Nullable image,
                                 NSInteger httpResponseCode,
                                 NSURL* url))completion {
   image_fetcher_->DownloadImage(
-      net::GURLWithNSURL(url), maxImageSize,
+      net::GURLWithNSURL(url), 256, 256,
```

Comment:
This will just ignore images over 256x256 instead of resize them, correct?

Reply:
This is the max SVG size.
1. If the image is > 256, it will still download it and return it.
2. If the image is SVG and > 256, it will download it, and return it at its original size.
3. If the image is SVG and does NOT specify a size or viewBox, it will automatically size it to 256x256 and return it.

Reply:
This is the regular `downloadImage` method though, so it would affect all of them, not just svg?

Reply:
Maybe best to drop a comment like what you put above these hard-coded values though

Reply:
> Maybe best to drop a comment like what you put above these hard-coded values though

It won't affect all images :) All images use `CGFLOAT_MAX` but the download method itself has parameters `svg_max_width` and `svg_max_height` and will only limit SVG to those dimensions

Reply:
Okay cool, thats fine, can you add a comment near the hardcoded values to explain them a bit